### ISSUE TITLE
Batch Size Error

### ISSUE THREAD

**Issue by haoyudong-97** [airsplay/VisualRelationships#2]
https://github.com/airsplay/VisualRelationships/blob/ca3214d8fe68d032ee620d3334b7a541ba57b626/src/speaker.py#L266-L269

Line 269 will lead to an error if I set batch_size to 1 (if I want to see the output sent from a particular pair), but this should be an easy fix.

**Comment by airsplay** [comment]
Great to know that! I would like to change it to `logits.squeeze(1)` to fix this issue. Does it sound correct to you?

**Comment by haoyudong-97** [comment]
Yep. Really nice work :)
